src/components/PersonItem.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link, useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={person.slug === slug ? 'has-background-warning' : ''}
+    >
+      <td>
+        <Link
+          to={`${person.slug}`}
+          className={person.sex === 'f' ? 'has-text-danger' : ''}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {!person.mother ? (
+          person.motherName || '-'
+        ) : (
+          <Link to={person.mother.slug} className="has-text-danger">
+            {person.motherName}
+          </Link>
+        )}
```

Comment:
The logic for displaying the mother's name seems to be incorrect. The condition `!person.mother` should likely be `person.mother` to check if the mother exists. If `person.mother` is truthy, you should display the link; otherwise, display `person.motherName || '-'`. This logic should be reversed to correctly render the link when the mother exists.